### PR TITLE
Update parameters.py

### PR DESCRIPTION
**Pull request by valdebrutal**
No need to loop over the keys, just fetch from dict.

### CODE REVIEW

File: utils/parameters.py [hunk 1]
```diff
@@ -14,8 +14,4 @@ def __init__(self):
         self._parameters_file = yaml.load(stream)
 
     def get_parameter(self, parameter):
-        for key, value in self._parameters_file.iteritems():
-            if key == parameter:
-                return value
-
-        raise Exception('Key %s not found' % parameter)
\ No newline at end of file
+        return self._parameters_file[parameter]
```
Comment (Psramos) [inline utils/parameters.py]: And what about the exception handling? 🤔

**Comment by intael** [comment]
You weren't handling it, just raising it. In this case it will raise too whenever the key doesn't exist in the dict.

File: utils/parameters.py (hunk 1 above)
Comment (intael) [reply utils/parameters.py]: You weren't handling it, just raising it. In this case it will raise too whenever the key doesn't exist in the dict.

Comment (intael) [reply utils/parameters.py]: 👺

Comment (Psramos) [reply utils/parameters.py]: Okey. Merging. In future projects I'll need to come back to this class so better to have it updated 🐘

**Review by Psramos — APPROVED** [review]

**Pull request merged**